SpatialGDK/Source/SpatialGDKFunctionalTests/Private/TestMapGeneration.cpp

```diff
@@ -38,6 +40,13 @@ bool GenerateTestMaps()
 		}
 	}
 
+	// Early out if we failed to delete the directory because otherwise the editor will
+	// spam popups about levels with the same name already existing.
+	if (!bSuccess)
+	{
+		return false;
+	}
```

Comment:
I guess this would also stop if the config folder failed to be deleted?

Reply:
Yes. I wasn't sure if that's necessary, but there's a high chance that generating the config will then fail if something was locking a previous file.